src/AutoRest.CSharp/LowLevel/Output/OperationMethodChainBuilder.cs

```diff
@@ -98,34 +96,20 @@ private bool ShouldConvenienceMethodGenerated(ReturnTypeChain returnTypeChain)
                 || !returnTypeChain.Convenience.Equals(returnTypeChain.Protocol));
         }
 
-        private bool IsParameterTypeSame(Parameter? first, Parameter? second)
-        {
-            if ((first == null && second != null)
-                || (first != null && second == null))
-            {
-                return true;
-            }
-
-            if (first == null && second == null)
-            {
-                return true;
-            }
-
-            return first!.Type.Equals(second!.Type);
-        }
+        private static bool IsParameterTypeSame(Parameter? first, Parameter? second) => first == null || second == null || first.Type.Equals(second.Type);
```

Comment:
So here if `first == null` and `second != null`, the two parameters are the same?

Reply:
That was the old behavior. Probably the name of the method isn't the good one.